{"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"other"}],"externalID":"s:10DigiMeCore8SDKErrorO5otheryA2CmF","modules":[{"name":"DigiMeCore"}],"title":"SDKError.other","role":"symbol","roleHeading":"Case","symbolKind":"case"},"identifier":{"url":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/other","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"other"}],"platforms":["iOS"]}],"kind":"declarations"}],"schemaVersion":{"patch":0,"major":0,"minor":3},"hierarchy":{"paths":[["doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore","doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"]]},"variants":[{"paths":["\/documentation\/digimecore\/sdkerror\/other"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","sections":[],"abstract":[{"type":"text","text":"An unexpected error has occurred - please contact support"}],"references":{"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore":{"title":"DigiMeCore","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore","abstract":[],"kind":"symbol","type":"topic","role":"collection","url":"\/documentation\/digimecore"},"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore/SDKError":{"title":"SDKError","type":"topic","abstract":[{"type":"text","text":"SDK Errors"}],"role":"symbol","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"SDKError","kind":"identifier"}],"url":"\/documentation\/digimecore\/sdkerror","navigatorTitle":[{"text":"SDKError","kind":"identifier"}],"kind":"symbol"},"doc://me.digi.sdk.example.librarycore.DigiMeCore/documentation/DigiMeCore/SDKError/other":{"role":"symbol","url":"\/documentation\/digimecore\/sdkerror\/other","identifier":"doc:\/\/me.digi.sdk.example.librarycore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/other","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"other","kind":"identifier"}],"title":"SDKError.other","abstract":[{"text":"An unexpected error has occurred - please contact support","type":"text"}],"type":"topic"}}}